front/src/containers/Notification/style.js

```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components'
+import { StyleSheet } from 'react-native'
+
+export const GlobalStyle = StyleSheet.create({
+  Icon: {
+    zIndex: 1,
+    position: 'absolute',
+    left: 65,
+    top: 60,
+  }
+})
+
+export const Container = styled.SafeAreaView`
+  flex: 1;
+`
+export const ContainerList = styled.TouchableOpacity`
+  flex-direction: row;
+  width: 100%;
+  margin: 1% 0;
+  padding: 3%;
+  align-items: center;
+`
+export const Reminder = styled.View`
+  background: #0311FC;
+  width: 30px;
+  height: 30px;
+  border-radius: 50px;
```

Comment:
To do a circle, you need set the border radius of component the half of square size. So, switch this to 15px